component/custom/custom_loader.go

```diff
@@ -0,0 +1,67 @@
+package custom
+
+import (
```

Comment:
As a good coding practice we have always created a test file for each component in the same directory. I do not see a UT file here for custom. This will be flagged in our code coverage pipelines and it will fail.

Reply:
I gave a thought about adding a unit test but couldn't come up something that made sense for UT. Any ideas around how to test an init() that is just registering new component without having a blobfuse mount. ?
I have added an E2E which expects you two have a blobfuse mount and validates read/write on that mount path.